end_to_end/1-data-prep-e2e.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>Even after going through this notebook, it is still not clear to me what's the objective of data wrangling. It looks that <code>customers</code>  and <code>claims</code>  are merged into one dataset. For understandability, I think it is better to state the objective in the beginning of the notebook. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>The data wrangling actually happens in Data Wrangler. I will clarify that thanks! Good point.</p>

Reply:
<p>I have now explained this in the very beginning of nb 1</p>